pom.xml

```diff
@@ -8,13 +8,13 @@
    <modelVersion>4.0.0</modelVersion>
    <groupId>org.jboss.forge.addon</groupId>
    <artifactId>wildfly-swarm</artifactId>
-   <version>2017.12.0-SNAPSHOT</version>
+   <version>2017.11.0</version>
```

Comment:
The version should still be SNAPSHOT. I'll perform a mvn release once the deps are ok

Reply:
Thanks!